.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: python
 python:
     - 2.7
+    - 3.4
+    - 3.6
```

Comment:
I can add it.
I didn't put it because I didn't have a python 3.5 environment on my machine, so I didn't test it myself. But travis is here for that!